CIDER/cider.py

```diff
@@ -1421,7 +1422,13 @@ def _get_scaffold(
             structure_list.clear()
             graph_framework_list = []
             for mol in framework_list:
-                graph_framework_list.append(MurckoScaffold.MakeScaffoldGeneric(mol))
+                try:
+                    graph_framework_list.append(MurckoScaffold.MakeScaffoldGeneric(mol))
+                except AtomValenceException:
+                    logger.info(
+                        "Scaffold can not be generated, molecule will be excluded from scaffold analysis!"
```

Comment:
I also agree with that. We can close this pull request once it is sorted out.
@hannbus What do you think?

Reply:
In what form should the identity be added? It could be the index or the scaffold, framework or the original molecule as string identifier or any other idea you suggest.

Reply:
I guess the id of the original molecule together with the exact description of where it caused an error would be best, if possible.

Reply:
I added the id and the index, as I needed the index for getting the id.